spec/barbeque/executor_spec.rb

```diff
@@ -19,11 +19,22 @@
     end
 
     it 'performs a job' do
-      expect(test_job).to receive(:perform).with(*args)
+      expect(test_job).to receive(:perform).with('foo', 'bar')
```

Comment:
It's inconsistent to have L5 too. If you prefer writing directly, rewrite other places to do so too.